File: JavaScript/101.js
function main()
{
    for (var multiplicand = 1; multiplicand <= 10; ++multiplicand)
    {
        for (var multiplicator = 1; multiplicator <= 10; ++multiplicator)
        {
            var result = multiplicator * multiplicand;
            write(multiplicator + " x " + multiplicand + " = " + result + "\n")
        }
        write("\n")
    }
}


main()

File: Python/101.py
#!/usr/bin/python

import sys

def main():
    multiplicand = 1
    while multiplicand <= 10:
        multiplicator = 1
        while multiplicator <= 10:
            result = multiplicator * multiplicand
            sys.stdout.write("%d x %d = %d\n" % (multiplicator, multiplicand, result))
            multiplicator += 1
        sys.stdout.write("\n")
        multiplicand += 1


main()

